src/app/components/itinerary/itinerarylist/itinerarylist.component.ts

```diff
@@ -1,6 +1,8 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, Input, OnInit } from '@angular/core';
 import { Itinerary } from 'src/app/models/itinerary';
 import { ItineraryService } from 'src/app/services/itinerary.service';
+import { Router, ActivatedRoute, ParamMap } from '@angular/router';
+import { TokenService } from 'src/app/services/token.service';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ispp-yourney/angular-app/pullRequest?prid=7157930)
```suggestion
import { TokenService } from "src/app/services/token.service";
```